apps/web/src/components/workflow/preview/in-app/v2/InboxPreviewContent.tsx

```diff
@@ -22,6 +23,31 @@ export const INBOX_TOKENS = {
   'Inbox/margin/message/txt/buttons': '1rem',
 } as const;
 
+const parseText = (text?: string) => {
```

Comment:
Nit-picky comment, we are rendering the markdown text, not parsing it ;)

```suggestion
const renderText = (text?: string) => {
```